backend/routes/graph.py

```diff
@@ -186,7 +186,7 @@ async def indented_concept_list_post(
         raise e
 
 
-def get_missing_in_between_nodes(G, nodes):
+def get_missing_in_between_nodes(G: nx.DiGraph, nodes: Iterable[Any]) -> Set[Any]:
```

Comment:
## Question/Suggestion: What after `get_missing_in_between_nodes()`?
We get a list of "missing in-betweens" from this, but it looks like next step would be to connect them to the graph.

Suggestion: What about instead having a function that returns (i) subgraph of everything connected and (ii) the list of missing in betweens? Intuitively seems better than me but I could be wrong.